### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Antime02**
This is the game of Random dice rolling where you can carry Sprig as a dice in your pocket and Play Game of  Dares. In this game you have to prepare a bowl of Dares and write numbers on the chits , now you can use you Sprig and roll the dice to give each other the dares and have fun.

# Your checklist for this pull request

**Author:** Harsh Kumar

**What is your game about?**

This game is about rolling the dice and playing the game of dares with your friends , you basically prepare the bowl of chits with dares written on them, you also have to write a no. on that chit upto 6. Now get you Sprig roll the dice and enjoy with your friends.

**How do you play your game?**

So basically what you have to do is upload this game to you Sprig console and after that when you are sitting with your friends , each on of you have to press 'S' Button this will show a no of dots on the screen based on the no. of dots you have to give each other the dares

## Code
Check off the items that are true. 
- [x] The game was made using the [Sprig editor](https://sprig.hackclub.com/editor).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [ ] The name of the image matches the name of your file. 

> Thanks for your PR!

### CODE REVIEW

**Comment by Antime02** [comment]
Please do check you my game Me and my friends have really worked on this project and also want to run this game on you professional console specially made for it

**Review by LucasHT22 — CHANGES_REQUESTED** [review]
Thanks for your PR! Where did you hear about Sprig?

**Comment by LucasHT22** [comment]
> Please do check you my game Me and my friends have really worked on this project and also want to run this game on you professional console specially made for it

We check all PRs:)

**Comment by Antime02** [comment]
My friends Ansh, Atman and Soumay asked me to contribute as a co-leader in their hack club named "Webventures" where we started our first project with sprig and spread awareness at our school jiva public school. We gave a lecture on how to build game in editor for sprig on 10th june 2023, so a curiosity sparked in back of my mind that i want a sprig

**Comment by LucasHT22** [comment]
Nice! Out of curiosity, did you use any game for inspiration?

**Comment by Antime02** [comment]
Hey can you please elaborate your question like what do you mean by "inspiration"

**Comment by Antime02** [comment]
so that I can revert you in best way possible

**Comment by Antime02** [comment]
Hey Lucas! I hope you are doing good Please reply to my question

**Comment by LucasHT22** [comment]
I thought i already answered, sorry. I mean used as a base

**Comment by Antime02** [comment]
No I have not used any game for inspiration but yes my co-leaders while introducing others about the hackclub in our school , we did use some of the games made by us to make more students interested in this program and make them join our club

**Comment by karamassie** [comment]
@LucasHT22 is this PR not passing the originality test or ?

File: games/dicee_dare.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+setMap(levels[level]);
+setPushables({
+  [player]: []
+});
+
+onInput("s", () => {
+
+    if (!gameStarted) {
+    console.log("Game started! Press 's' to roll the dice and 'd' to reset.");
```
Comment (LucasHT22) [inline games/dicee_dare.js:167]: @Antime02 could you use addText() instaed of console.log?

Comment (Antime02) [reply games/dicee_dare.js:167]: how do I edit the code and resubmit it like do I have to Close the pull Request and then open it again

Comment (LucasHT22) [reply games/dicee_dare.js:167]: Sorry for the delay! nope, go to `Files changed` > `...` > `edit`

Comment (Antime02) [reply games/dicee_dare.js:167]: sorry Lucas but where do I find this "File changed"

**Comment by Antime02** [comment]
Hey Lucas! I edited the file kindly check

**Comment by LucasHT22** [comment]
What about add 2-3 words per addText? :) To avoid this:
image

**Comment by Antime02** [comment]
do check now , is it fine ?

**Comment by LucasHT22** [comment]
I think it's better to keep the cases empty to who is playing complete... the actual isnt appropriate... if you understand me

**Comment by Antime02** [comment]
Sorry but i am not able to understand , just tell me what i need to do now

**Comment by LucasHT22** [comment]
I'll be direct, some of these are not appropriate, especially d5, please change
image

**Comment by Antime02** [comment]
do check now

**Comment by LucasHT22** [comment]
Right:) Out of curiosity, some friend helped you with code?

**Comment by Antime02** [comment]
nope no one really helped me in making my game but yes I did consulted my teacher who knew about JavaScript

**Review by LucasHT22 — APPROVED** [review]
Alright, Congrats:)
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

Hack Clubbers would love to see this! Just post a link and 1-2 sentences talking about your game to our [#ship](https://hackclub.slack.com/archives/C0M8PUPU6) [channel](https://hackclub.com/slack/):)

**Pull request merged**